lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestTaskExceptions.cs

```diff
@@ -240,7 +240,7 @@ public byte[] Call(byte[] memento)
                     case TaskManager.TaskGroupCommunicationError:
                         throw new IMRUTaskGroupCommunicationException(_taskExceptionMessage);
                     default:
-                        throw new IMRUTaskSystemException(_taskExceptionMessage, new SystemException(InnerExceptionMessage));
+                        throw new IMRUTaskSystemException(_taskExceptionMessage, new Exception(InnerExceptionMessage));
```

Comment:
Use a particualt exception type, such as REEFSystemException. And then check the same exception in test Assert statement.